framework/include/constraints/ConstraintWarehouse.h

```diff
@@ -45,6 +46,7 @@ class ConstraintWarehouse : public MooseObjectWarehouse<Constraint>
    */
   const std::vector<MooseSharedPointer<NodalConstraint> > & getActiveNodalConstraints() const;
   const std::vector<MooseSharedPointer<FaceFaceConstraint> > & getActiveFaceFaceConstraints(const std::string & interface) const;
+  const std::vector<MooseSharedPointer<ElemElemConstraint> > & getActiveElemElemConstraints(unsigned int interface_id) const;
```

Comment:
We might have different interfaces to be constrained differently. This info is not from mesh. The ```interface id``` should be generated by ```XFEM``` or somehow specified in the ```XFEM``` block but we have not implemented this capability yet.